src/helpers/Constants.js

```diff
@@ -17,6 +17,12 @@ export function getAndroidVersion() {
   return isAndroid ? parseInt(Platform.Version, 10) : undefined;
 }
 
+export function getSafeAreaInsets(mode) {
+  return (mode === 'landscape') ? 
+    {paddingLeft: 44, paddingRight: 44, paddingBottom: 24, paddingTop: 0} : 
```

Comment:
I would change the names here to something more general like top, left, right, bottom instead of padding. 
cause they can be used for all kind of scenarios.

Reply:
Ok.